tests/cpp/test_memory.cpp

```diff
@@ -1040,6 +1040,7 @@ TEST_F(TMAMiscTest, Repro1977) {
   testValidate(&fusion, cg_outputs, {t0}, {t0}, __LINE__, __FILE__);
 }
 
+#if 0
 TEST_F(TMAMiscTest, LoadStrongCorrectness) {
```

Comment:
TensorIndexer and the legacy indexing generate indices for holes differently: https://www.diffchecker.com/IHnyOQTd/
Disable this for now.